go/extensions/types.go

```diff
@@ -1,12 +1,75 @@
 package extensions
 
 import (
-    sdk "github.com/cosmos/amino-js/go/lib/cosmos/cosmos-sdk/types"
+	bridge "github.com/cosmos/amino-js/go/lib/bridge/types"
+	sdk "github.com/cosmos/amino-js/go/lib/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/amino-js/go/lib/tendermint/tendermint/libs/common"
+	tm "github.com/cosmos/amino-js/go/lib/tendermint/tendermint/types"
 )
 
 var _ sdk.Msg = (*TxCreateMarket)(nil)
```

Comment:
`TxCreateMarket` can be removed right?

Reply:
https://github.com/denalimarsh/amino-js/pull/2/commits/cecdc355c128f39ca475006791031e59fdb102a3